server/config/config.template.yaml

```diff
@@ -40,6 +40,12 @@ server:
   ### Public URL. It is used in welcome.html if not configured it will be taken from 'Host' http header on welcome.html requests
   #public_url: https://yourhost #Optional.
 
+  ### Limit the count of columns for any destinations.
+  ### It helps to prevent generating an infinite amount of columns in databases.
+  ### Zero value turns off the column count calculation.
+  ### It can be overridden at the destination level only by a positive value.
+  max_columns: 100 # Optional
```

Comment:
We shouldn't have global value for `max_columns`

Reply:
* `100` in the code - it looks like the magic number and it cannot be changed.
* Configuration settings provide a better description and provide the capability to change it.
* There is the capability to make `self-hosted` installation.
It is better to use one single option than several options, if admins want to change such setting.